test/test_sparse.py

```diff
@@ -2363,6 +2363,304 @@ def test_sparse_to_numpy(self):
         t = torch.sparse_coo_tensor(torch.tensor(([0, 0], [2, 0])), torch.tensor([1, 4]))
         self.assertRaises(TypeError, lambda: t.numpy())
 
+    def test_softmax(self):
```

Comment:
You could use `@cpu_only` decorator here

Reply:
yes, fixed.